package.json

```diff
@@ -8,7 +8,7 @@
     "lint": "eslint --ext .js",
     "server-test": "mocha -R spec --recursive",
     "browser-test": "karma start",
-    "test": "npm run server-test && npm run browser-test"
+    "test": "npm run server-test"
```

Comment:
See the comment below; this never worked, and I am totally fine adding it, but it's for a separate PR and development time.